jbake-core/src/main/java/org/jbake/app/configuration/DefaultJBakeConfiguration.java

```diff
@@ -84,6 +92,22 @@ private String getAsString(String key, String defaultValue) {
         return compositeConfiguration.getString(key, defaultValue);
     }
 
+    /**
+     * Handle invalid or unavailable charset.
+     */
+    private Charset getAsCharset(String key, Charset defaultCharset)
+    {
```

Comment:
Please don't mix styles. The unwritten convention is placing the bracket on the same line as the method definition or controll statement.

Yes:

```
if ( foo ) {
  return bar;
}
```

No:

```
if ( foo )
{
   return bar;
}
```

Reply:
Ok I will align the styles. I blame IDEA. Or my little finger twitching above enter :)